api/app/tests/apps/api/v1/json_schema/get_signals_v1_public_questions_categories_{slug}_sub_categories_{sub_slug}.json

```diff
@@ -0,0 +1,75 @@
+{
```

Comment:
For some reason the endpoint is not showing up in the spec yet: http://localhost:8000/signals/v1/

Reply:
The route is not defined by a router, it is added manually. Therefore it is not showing up in the generated API spec. However when there is time this is something that we need to figure out. A couple of endpoints are missing here because of this.